src/pages/components/text-inputs.mdx

```diff
@@ -0,0 +1,199 @@
+
+import FauxInput from '../../examples/text-inputs/FauxInput.tsx';
+import FauxInputCode from '!!raw-loader!../../examples/text-inputs/FauxInput.tsx';
+
+<CodeExample code={FauxInputCode}>
+  <FauxInput />
+</CodeExample>
+
+### Regular weight label
+
+Text input labels are emphasized by default. They can be regular weight using the `isRegular`
+property.
+
+Regular labels are not recommended for text inputs, as they flatten visual hierarchy making
+scanning more difficult.
+
+import RegularWeightLabel from '../../examples/text-inputs/RegularWeightLabel.tsx';
+import RegularWeightLabelCode from '!!raw-loader!../../examples/text-inputs/RegularWeightLabel.tsx';
+
+<CodeExample code={RegularWeightLabelCode}>
+  <RegularWeightLabel />
+</CodeExample>
+
+### Hint
+
+Hint adds additional context to the text Input’s label.
+
+import Hint from '../../examples/text-inputs/Hint.tsx';
+import HintCode from '!!raw-loader!../../examples/text-inputs/Hint.tsx';
+
+<CodeExample code={HintCode}>
+  <Hint />
+</CodeExample>
+
+### Size
+
+A text Input can be default or compact.
+
+import Size from '../../examples/text-inputs/Size.tsx';
+import SizeCode from '!!raw-loader!../../examples/text-inputs/Size.tsx';
+
+<CodeExample code={SizeCode}>
+  <Size />
+</CodeExample>
+
+### Bare
+
+Input borders and padding can be removed using the `isBare` property. Use an
+[aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute)
+to label the input.
+
+import Bare from '../../examples/text-inputs/Bare.tsx';
+import BareCode from '!!raw-loader!../../examples/text-inputs/Bare.tsx';
+
+<CodeExample code={BareCode}>
+  <Bare />
+</CodeExample>
+
+### Focus inset
+
+Box-shadow focus styling can be inset using the `focusInset` property.
+
+This keeps the focus contained within the input area when it is flush with another component.
+
+import FocusInset from '../../examples/text-inputs/FocusInset.tsx';
+import FocusInsetCode from '!!raw-loader!../../examples/text-inputs/FocusInset.tsx';
+
+<CodeExample code={FocusInsetCode}>
+  <FocusInset />
+</CodeExample>
+
+### Placeholder
+
+Placeholder text gives an example of content format. See [Placeholder text](#TODO) standard for
+best practices.
+
+import Placeholder from '../../examples/text-inputs/Placeholder.tsx';
+import PlaceholderCode from '!!raw-loader!../../examples/text-inputs/Placeholder.tsx';
+
+<CodeExample code={PlaceholderCode}>
+  <Placeholder />
+</CodeExample>
+
+### Validation
+
+Success, warning, or danger validation messages are shown using the Message component with the text
+Input.
+
+import Validation from '../../examples/text-inputs/Validation.tsx';
+import ValidationCode from '!!raw-loader!../../examples/text-inputs/Validation.tsx';
+
+<CodeExample code={ValidationCode}>
+  <Validation />
+</CodeExample>
+
+### Inline
+
+An Input uses `display:block` by default. It can use `display:inline`. See example code for inline
+styling overrides.
```

Comment:
This sentence isn't accurate.

- a `Field` is `display: block` by default
- an `Input` is `display: inline-block` by default
- the associated demo is showing how a label can be inlined with an input via `display: flex`

Reply:
This concern was brought up yesterday in the Google doc. How does the following sound?

> An Input can be styled to have an inline layout. See example code for inline styling overrides.

If that copy is acceptable, then I can update this mdx file and make adjustments to the Google doc accordingly.

Reply:
Update: Spoke with @rossmoody and we've punted this section.